### PR TITLE
Alternative method for adding internal certificates to Chrome and Firefox

### PR DESCRIPTION
**Pull request by luisfcorreia**
Alternative method for adding internal certificates to Chrome and Firefox

### Motivation and Context
Improve documentation fro certificate handling

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist
- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1074,7 +1074,51 @@ RUN mkdir -p -m755 /home/seluser/.pki/nssdb \ #create nssdb folder
...
+___
+
+## Alternative method: Add certificates to existing Selenium based images for browsers
+
+As an alternative, you can add youe certificate files to existing Selenium images. This practical example
+assumes you have a known image to use as a build image and have a way to publish new images to your local
+docker registry.
+
+This also assumes you are using a redhat based image as *build* _with_ your internal CA already added.
+(default location for rocky linux is /etc/pki/ca-trust/source/anchors/YOUR_CA.pem)
```
Comment (diemol) [inline README.md]: The issue I see is that users will ask how to build this image. Does it need to be RedHat? Could it be any Linux based image?

File: README.md [hunk 2]
```diff
@@ -1074,7 +1074,51 @@ RUN mkdir -p -m755 /home/seluser/.pki/nssdb \ #create nssdb folder
...
+
+(tested working 2023-05-17, based on images from Selenium 4.9.1 release)
```
Comment (diemol) [inline README.md]: This is not really needed, we can always troubleshoot or tell people to use latest.

File: README.md [hunk 3]
```diff
@@ -1074,7 +1074,51 @@ RUN mkdir -p -m755 /home/seluser/.pki/nssdb \ #create nssdb folder
...
+# Get a standard image for creating nssdb file
+FROM rockylinux:8.6 as build
+RUN yum install -y nss-tools
+RUN mkdir -p -m755 /seluser/.pki/nssdb \
+    && certutil -d sql:/seluser/.pki/nssdb -N --empty-password \
+    && certutil -d sql:/seluser/.pki/nssdb -A -t "C,," -n YOUR_CA -i /etc/pki/ca-trust/source/anchors/YOUR_CA.pem \
+    && chown -R 1200:1201 /seluser
 
+# Start from Selenium image and add relevant files from build image
+FROM selenium/node-chrome:4.9.1
```
Comment (diemol) [inline README.md]: Full tag including date, please

File: README.md [hunk 4]
```diff
@@ -1074,7 +1074,51 @@ RUN mkdir -p -m755 /home/seluser/.pki/nssdb \ #create nssdb folder
...
+Example for Firefox:
+```
+# Get a standard image for working on
+FROM rockylinux:8.6 as build
+RUN mkdir -p "/distribution" "/certs" && \
+    cp /etc/pki/ca-trust/source/anchors/YOUR_CA*.pem /certs/ && \
+    echo '{ "policies": { "Certificates": { "Install": ["/opt/firefox-latest/YOUR_CA.pem"] }} }' >"/distribution/policies.json"
+
+# Start from Selenium image and add relevant files from build image
+FROM selenium/node-firefox:4.9.1
```
Comment (diemol) [inline README.md]: Full tag including date, please

**Review by diemol — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (diemol) [reply README.md]: What I mean is... would it be possible to add the instructions to build that image?

File: README.md [hunk 5]
```diff
@@ -1074,7 +1074,51 @@ RUN mkdir -p -m755 /home/seluser/.pki/nssdb \ #create nssdb folder
     && pk12util -d sql:/home/seluser/.pki/nssdb -i client_cert.p12 -W password_of_clent_cert # client certificate install
 ```
 This way the certificates will be installed and the node will start automatically as before.
+___
+
+## Alternative method: Add certificates to existing Selenium based images for browsers
+
+As an alternative, you can add youe certificate files to existing Selenium images. This practical example
```
Comment (krmahadevan) [inline README.md]: Typo in `your` (its spelt as `youe`)

File: README.md (hunk 1 above)
Comment (luisfcorreia) [reply README.md]: It can be any image, the actual instructions may vary.
i'll clarify this

File: README.md (hunk 2 above)
Comment (luisfcorreia) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (diemol) [reply README.md]: An example is always more helpful

**Review by diemol — APPROVED** [review]

**Pull request merged**